client-course-schedulizer/src/utilities/services/facultySchedule.ts

```diff
@@ -0,0 +1,16 @@
+import { Schedule } from "../interfaces/dataInterfaces";
+
+// Get list of unique professors.
+export const getProfs = (schedule: Schedule): string[] => {
+  const professors: string[] = [];
+  schedule.courses.map((course) => {
+    return course.sections.map((section) => {
```

Comment:
Change these to `forEach` since you're not doing anything with the returned array from `map`

Reply:
Good call. Thanks